app/assets/stylesheets/modules/shared.css

```diff
@@ -222,46 +222,10 @@ span.github-btn {
   font-weight: 800;
 }
 
-.adoption__tag {
-  display: inline-block;
-  font-size: 0.8em;
-  font-weight: 800;
-  color: #e9573f;
-  border: 1px solid #e9573f;
-  border-radius: 8px;
-  padding: 0.25em 1em;
-  margin-bottom: 0.9em;
-  vertical-align: 0.3em;
-  transition: color 0.25s ease-in,
-    background-color 0.25s ease-in,
-    transform 0.3s ease-out;
-}
-
-.adoption__tag:hover{
-  color: white;
-  background-color: #e9573f;
-}
-
 .adoption__heading {
```

Comment:
why is this not being removed?

Reply:
It’s used elsewhere. Probably it should be renamed.

Reply:
:information_source: renamed to description_heading (since it is used in `dt` tags)

Reply:
@martinemde this is missing now also